lib/packer.rb

```diff
@@ -15,12 +15,20 @@ def instance
     @instance ||= Packer::Instance.new
   end
 
+  def rails?
+    defined? Rails
+  end
+
+  def sinatra?
+    defined? Sinatra
+  end
+
   delegate :logger, :logger=, :env, to: :instance
   delegate :compile, :compiler, :manifest, :commands, :config, :dev_server, to: :instance
   delegate :bootstrap, :clobber, :compile, to: :commands
 end
 
-require 'packer/version'
```

Comment:
Why was `packer/version` removed from here?

Reply:
Because it is already required in `packer.gemspec` line 6.